guides/release/components/built-in-components.md

```diff
@@ -1,190 +1,215 @@
+The event name (e.g. `"focusout"`, `"input"`, `"keydown"`) always follows the casing that the HTML standard uses.
 
-* `@enter`
-* `@insert-newline`
-* `@escape-press`
-* `@focus-in`
-* `@focus-out`
-* `@key-down`
-* `@key-press` ([Deprecated Web API](https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event))
-* `@key-up`
+Due to legacy code, it is possible to call an action by passing an event argument.
 
+```handlebars
+<label for="input-name">Name:</label>
+<Input
+  @id="input-name"
+  @value={{this.name}}
+  @input={{this.validateName}}
+/>
+```
 
-More [event types](https://api.emberjs.com/ember/release/classes/Component#event-handler-methods) are also supported but these events need to be written in camelCase format, such `mouseEnter`. Note, there are events of the same type in both the list above and linked. Event names listed above must be dasherized. Additional work is performed on these events.
+The argument name is always dasherized (e.g. `@focus-out`, `@input`, `@key-down`). To minimize confusion, we recommend that you use the `{{on}}` modifier. ([Learn more about the `{{on}}` modifier.](../../upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier))
 
-### Checkboxes
+Lastly, Ember Classic provided custom input events `@enter` and `@escape-press`. These are not a part of the HTML standard, but may facilitate accessibility.
 
-You can also use the
-[`<Input />`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input)
-component to create a checkbox by setting its `@type`:
+Starting with Ember Octane, please consider [writing a modifier](../../upgrading/current-edition/glimmer-components/#toc_writing-your-own-modifiers) to separate concerns: The component manages the state, while the modifier manages interactions with the DOM. If you prefer installing an addon, we recommend the `{{on-key}}` modifier from [ember-keyboard](https://github.com/adopted-ember-addons/ember-keyboard). Your action will receive an actual `event` object.
 
 ```handlebars
-<label for="admin-checkbox">Is Admin?</label>
-<Input id="admin-checkbox" @type="checkbox" name="isAdmin" @checked={{this.isAdmin}} />
+{{!-- Before --}}
+<Input
+  @enter={{this.doSomething}}
+  @escape-press={{this.doSomethingElse}}
+/>
+
+{{!-- After --}}
+<Input
+  {{on-key "Enter" this.doSomething}}
+  {{on-key "Escape" this.doSomethingElse event="keydown"}}
+/>
 ```
 
-Checkboxes support the following properties:
+Note, the `keydown` event was used for `Escape` because `keypress` is deprecated.
 
-* `@checked`
-* `disabled`
-* `tabindex`
-* `indeterminate`
-* `name`
-* `autofocus`
-* `form`
 
+### Checkboxes
 
-Which can be bound or set as described in the previous section.
+You can use the
+[`<Input>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input)
+component to create a checkbox. Please set `@type` to the string `"checkbox"`, and use `@checked` instead of `@value`.
```

Comment:
```suggestion
component to create a checkbox. Set `@type` to the string `"checkbox"`, and use `@checked` instead of `@value`.
```

Just edits to match the tone of the rest of the guides.